topostats/io.py

```diff
@@ -122,7 +129,7 @@ def _spm_pixel_to_nm_scaling(self, channel_data) -> float:
         Parameters
```

Comment:
Similarly I missed it in previous code reviews but is there a type for the `channel_data`?